pkgs/modules/deno/default.nix

```diff
@@ -0,0 +1,32 @@
+{ pkgs, lib, ... }:
+
+let
+  inherit (pkgs) deno;
+  version = lib.versions.major deno.version;
+
+  extensions = [ ".json" ".jsonc" ".js" ".jsx" ".ts" ".tsx" ];
+in
+
+{
+  id = "deno-${version}";
+  name = "Deno Tools";
+
+  replit.packages = [
+    deno
+  ];
+
+  replit.runners.deno-script-runner = {
+    name = "deno";
+    language = "javascript";
+    inherit extensions;
+    fileParam = true;
+    start = "${deno}/bin/deno run $file";
```

Comment:
I wonder if we should allow all access by default

Reply:
so i decided not to, because that's contra the deno philosophy. if users want to have some perms enabled, then they should do so via `tasks` in their `deno.json`

Reply:
Mmm, maybe. For Replit specifically the environment is sandboxed that this might be overkill, it feels like we will frustrate people instead of delighting them. Couldn't people also use tasks to run sandboxed tasks?

Reply:
that's a great point! i'll go ahead and make the default `--allow-all`

Reply:
force-pushed with this change